src/main/java/io/jenkins/plugins/opentelemetry/semconv/JenkinsOtelSemanticAttributes.java

```diff
@@ -113,5 +113,8 @@
 
     public static final AttributeKey<String>        ELASTIC_TRANSACTION_TYPE = AttributeKey.stringKey("type");
 
-
+    /**
+     * The stage name for the executed test.
+     */
+    public static final AttributeKey<String>        TEST_STAGE_NAME = AttributeKey.stringKey("test.stage.name");
```

Comment:
@mdelapenya , What do you about this particular attribute? This will be stored in the JUnitAction and therefore could be consumed.

Reply:
To distinguish from a test case from a test suite?
```suggestion
    public static final AttributeKey<String>        TEST_STAGE_NAME = AttributeKey.stringKey("tests.stage.name");
```

Reply:
No, it's the stage name where the junit was executed from, for instance:

```
pipeline {
   ...
   stages {
      stage('unit-test') {
         steps {
            ..
            junit 'unit-test.xml'
         }
      }
      stage('perf-test') {
         steps {
            ..
            junit 'perf-test.xml'
         }
     }
}
```

Although, I cannot find a way to provide the context to the consumer of this particular action

Reply:
Yes, then it represents the context of a test execution: a list of test suites, then tests fits more to me 🙂

Reply:
I had the same issue with test Vs tests in the semconv package in the junit reporter